uiowa/uiowa
# [Image block] Added circle and no crop media shapes

Trying again on https://github.com/uiowa/uiowa/issues/6803. 

Resolves https://github.com/uiowa/uiowa/issues/6368
Resolves https://github.com/uiowa/uiowa/issues/6803

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/1776/layout
```

- Add `No crop - fit height` style to the image block on the page and verify that the image fills the entire container. 
- Add `Circle` style to the image block and very that the image is set to a square format with a circle around it
